src/node_url.h

```diff
@@ -159,9 +159,8 @@ class URL {
 
   std::string path() const {
     std::string ret;
-    for (auto i = context_.path.begin(); i != context_.path.end(); i++) {
-      ret += '/';
-      ret += *i;
+    for (const std::string& i : context_.path) {
```

Comment:
Maybe rename it to something like `path_element`?
To me, `i` means "index" and that's how it's always used in JS.